lib/features/evolution_tab/widgets/evolution_tile.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:pokedex/utilities/constants.dart';
+import 'package:pokedex/utilities/extensions.dart';
+
+class EvolutionTile extends StatelessWidget {
+  const EvolutionTile({
+    this.name,
+    this.id,
+    this.isNotFirstEvolution,
+  });
+
+  final String? name;
+  final int? id;
+  final bool? isNotFirstEvolution;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        if (isNotFirstEvolution ?? false)
+          (Padding(
+            padding: EdgeInsets.all(2),
+            child: Icon(
+              IconData(
+                0xe09c,
+                fontFamily: 'MaterialIcons',
+                matchTextDirection: true,
+              ),
+            ),
+          )),
+        Column(
+          children: [
+            Image.network(
```

Comment:
Try using this if its better:

'''
CachedNetworkImage(
    imageUrl: '$iconUrl${evolution?.id}.png',
    height: 100,
    width: 100,
    placeholder: (context, url) => const LoadingWidget(),
     errorWidget: (context, url, error) => const Icon(Icons.error),
 )
'''

Add `cached_network_image` in pubspec.yaml